src/tools/rust-analyzer/crates/hir-ty/src/layout/target.rs

```diff
@@ -2,7 +2,7 @@
 
 use base_db::CrateId;
 use hir_def::layout::TargetDataLayout;
-use ra_ap_rustc_abi::{AlignFromBytesError, TargetDataLayoutErrors};
```

Comment:
We should rename the ra_ap_rustc crates in the cargo toml dependencies to the in-tree ones instead of doing `extern crate` aliases to prevent this from happening again.

Reply:
Good idea, if possible. Now I'm wondering whether we build the `ap` crates when in-tree.

Reply:
We do buildthem. There is no way around that without making those be feature gated I think, so not really an option to not build them (i'd rather not add more features for this stuff)

Reply:
alternatively we should prefer extern crate aliases when searching for paths I guess :p